### PR TITLE
CI: Fixing the scripts

### PR DESCRIPTION
**Pull request by Tal-or**
- Wrong script name in the make target 
- Export all the variables so child processes will have them as well
- Having the test running with some default RTE_CONTAINER_IMAGE is usually useless since it doesn't contain the latest changes, but yet the CI will run and we might think that everything is ok.
The test will failed at the beginning if the RTE_CONTAINER_IMAGE won't be mention explicitly

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
this means the reviewer should be more careful next time.

**Comment by Tal-or** [comment]
/hold
Got a local cluster, let's run locally first.

**Comment by Tal-or** [comment]
/hold cancel
CI passed locally

**Comment by Tal-or** [comment]
@fromanirh CI passed locally as well

File: hack/run-e2e-kni.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ PROJECT_DIR="${BASE_DIR}"/..
 export RTE_NAMESPACE="${RTE_NAMESPACE:-rte-e2e}"
 export E2E_NAMESPACE_NAME="${E2E_NAMESPACE_NAME:-rte-e2e}"
 export E2E_TOPOLOGY_MANAGER_POLICY="${E2E_TOPOLOGY_MANAGER_POLICY:-single-numa-node}"
+export RTE_CONTAINER_IMAGE="${RTE_CONTAINER_IMAGE}"
```
Comment (fromanirh) [inline hack/run-e2e-kni.sh:14]: this seems redundant, but should not harm

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**